spec/presenters/tree_builder_belongs_to_hac_spec.rb

```diff
@@ -0,0 +1,110 @@
+describe TreeBuilderBelongsToHac do
+  before do
```

Comment:
I will. When I don't need them to create tree and be in relationship with each other :)